programs/uxd/src/instructions/lsd/initialize_lsd_depository.rs

```diff
@@ -0,0 +1,153 @@
+
+#[derive(Accounts)]
+pub struct InitializeLsdDepository<'info> {
+    /// #1 Authored call accessible only to the signer matching Controller.authority
+    pub authority: Signer<'info>,
+
+    /// #2
+    #[account(mut)]
+    pub payer: Signer<'info>,
+
+    /// #3
+    #[account(
+        mut,
+        seeds = [CONTROLLER_NAMESPACE],
+        bump = controller.load()?.bump,
+        has_one = authority @UxdError::InvalidAuthority,
+    )]
+    pub controller: AccountLoader<'info, Controller>,
+
+    /// #4 
+    #[account(
+        init,
+        seeds = [LSD_DEPOSITORY_NAMESPACE, collateral_mint.key().as_ref()], 
+        bump,
+        payer = payer,
+        space = LSD_DEPOSITORY_SPACE,
+    )]
+    pub depository: AccountLoader<'info, LsdDepository>,
+
+    /// #5
+    pub collateral_mint: Account<'info, Mint>,
+
+    /// #6
+    pub liquidation_mint: Account<'info, Mint>,
+
+    /// #7
+    #[account(
+        init,
+        seeds = [LSD_PROFITS_TOKEN_ACCOUNT_NAMESPACE, depository.key().as_ref()],
+        token::authority = depository,
+        token::mint = liquidation_mint,
+        bump,
+        payer = payer,
+    )]
+    pub profits_token_account: Account<'info, TokenAccount>,
+
+    /// #6
+    pub system_program: Program<'info, System>,
+
+    /// #7
+    pub token_program: Program<'info, Token>,
+
+    /// #8
+    pub rent: Sysvar<'info, Rent>,
+}
+
+pub(crate) fn handler(ctx: Context<InitializeLsdDepository>) -> Result<()> {
+    let depository_bump = *ctx
+        .bumps
+        .get("depository")
+        .ok_or_else(|| error!(UxdError::BumpError))?;
+    let profits_token_account_bump = *ctx
+        .bumps
+        .get("profits_token_account")
+        .ok_or_else(|| error!(UxdError::BumpError))?;
+
+    let redeemable_mint_unit = 10_u64
+        .checked_pow(
+            ctx.accounts
+                .controller
+                .load()?
+                .redeemable_mint_decimals
+                .into(),
+        )
+        .ok_or_else(|| error!(UxdError::MathError))?;
+
+    let depository = &mut ctx.accounts.depository.load_init()?;
+    depository.bump = depository_bump;
+    depository.version = LSD_DEPOSITORY_ACCOUNT_VERSION;
+    depository.collateral_mint = ctx.accounts.collateral_mint.key();
+    depository.collateral_mint_decimals = ctx.accounts.collateral_mint.decimals;
+    depository.liquidation_mint = ctx.accounts.liquidation_mint.key();
+    depository.liquidation_mint_decimals = ctx.accounts.liquidation_mint.decimals;
+    depository.profits_token_account = ctx.accounts.profits_token_account.key();
+    depository.profits_token_account_bump = profits_token_account_bump;
+
+    // Configuration
+    depository.borrowing_disabled = true;
+    depository.redeemable_amount_under_management_cap = DEFAULT_REDEEMABLE_UNDER_MANAGEMENT_CAP
+        .checked_mul(redeemable_mint_unit)
+        .ok_or_else(|| error!(UxdError::MathError))?;
+    depository.borrowing_fee_bps = 0;
```

Comment:
nit: `u8::MIN;` to match other initializations

Reply:
tyvm